### DISCUSSION TITLE
Download wird abgebrochen

### DISCUSSION THREAD

**Discussion by LOWL1** [albar965/littlenavmap#1209] [General]
ich versuche version 3.012 Downloden wird aber immer abgebrochen bitte un Hilfe

**Comment by albar965** [comment]
Gibt es eine genaue Fehlermeldung?

Versuche mal den Anti-Virus temporär auszuschalten, einen anderen Browser zu benutzen oder auf einen alternativen Download zuzugreifen: https://albar965.github.io/downloads.html

**Comment by LOWL1** [comment]
Danke für die rasche Antwort werde das versuchen ging bis 3.08 ok aber dann leider nichts.Dankefür das SUPER Programm ohne LNM kein IFR Flug bei mir .LG Erich aus Österreich

 

Von: Alexander Barthel ***@***.***> 
Gesendet: Freitag, 22. November 2024 17:57
An: albar965/littlenavmap ***@***.***>
Cc: LOWL1 ***@***.***>; Author ***@***.***>
Betreff: Re: [albar965/littlenavmap] Download wird abgebrochen (Discussion #1209)

 

Gibt es eine genaue Fehlermeldung?

Versuche mal den Anti-Virus temporär auszuschalten, einen anderen Browser zu benutzen oder auf einen alternativen Download zuzugreifen: https://albar965.github.io/downloads.html

**Comment by albar965** [comment]
Wenn der Browser unter Windows herumzickt, kann vielleicht das helfen:
https://albar965.github.io/littlenavmap-faq.html#windows-download
